doc/nrf/ug_matter_configuring_controller.rst

```diff
@@ -36,7 +36,7 @@ CHIP Tool for Linux or macOS
 ****************************
 
 CHIP Tool for Linux or macOS is the default implementation of the Matter controller role, recommended for the nRF Connect platform.
-You can use it either on a separate device from Thread Border Router or on the same device as Thread Border Router, depending on which :ref:`development environment setup option <ug_matter_configuring>` you choose.
+You can use it either on a device connected to the IPv6 network or on a separate device, depending on which :ref:`development environment setup option <ug_matter_configuring>` you choose.
```

Comment:
The controller must be run on a device connected to the IPv6 network. The only options we have is when testing a Thread device - then we can run the controller on the same device as OTBR or on a separate one. I would leave the previous sentence and only add info that this option applies only to testing Thread devices.